app/page/info/activity.tsx

```diff
@@ -1,3 +1,4 @@
+import { getChainName } from "~app/page/info/index"
```

Comment:
It would be better to put common util into package (maybe `packages/network/util`). We should keep jsx files to export only view functions.